src/install/dracut-install.c

```diff
@@ -49,6 +49,14 @@
 #include "util.h"
 #include "strv.h"
 
+#define _asprintf(strp, fmt, ...) \
+        do { \
+            if (dracut_asprintf(strp, fmt, __VA_ARGS__) < 0) { \
+                    log_err("Out of memory\n"); \
```

Comment:
log_error instead of log_err should be used.

Reply:
Thanks. Fixed it.